Cite Foote et al, JAMA Oncology, August 19, 2021. (#1017)
greenelab/covid19-review: content/07.pathogenesis.md

ACE2 and TMPRSS2 have been identified as the primary entry portal and as a critical protease, respectively, in facilitating the entry of SARS-CoV-1 and SARS-CoV-2 into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues, and organs are most likely to be infected by SARS-CoV-2.
ACE2 is expressed in numerous organs, such as the heart, kidney, and intestine, but it is most prominently expressed in alveolar epithelial cells; this pattern of expression is expected to contribute to the virus' association with lung pathology [@doi:10.1101/2020.02.24.963348; @doi:10.1007/s00134-020-05985-9; @doi:10.1016/j.phrs.2020.104833] as well as that of SARS [@doi:10.1002/path.1570].

> [@doi:10.1001/jamaoncol.2021.3585] reported indirect evidence from a retrospective observational study, suggesting that certain antineoplastic therapies such as gemcitabine may reduce risk of SARS-CoV-2 infection in patients with cancer, possibly via decreased ACE2 expression.

Reviewer: ```suggestion
A retrospective observational study reported indirect evidence that certain antineoplastic therapies, such as the chemotherapy drug gemcitabine, may reduce risk of SARS-CoV-2 infection in patients with cancer, possibly via decreased ACE2 expression [@doi:10.1001/jamaoncol.2021.3585].
```
